js/gemini.js

```diff
@@ -1366,10 +1362,11 @@ module.exports = class gemini extends Exchange {
             let payload = this.json (request);
             payload = this.stringToBase64 (payload);
             const signature = this.hmac (payload, this.encode (this.secret), 'sha384');
+            const authorizationKey = 'Bearer ' + this.token;
```

Comment:
Can you please elaborate a bit – where does this `token` come from ?

Reply:
We might have to implement a unified `signIn` method in order for the user to be able to obtain the auth `token` like we do in the other exchanges.

Reply:
The still show the same authentication schema on their API docs:

<img width="1166" alt="Screenshot 2022-06-01 at 02 50 57" src="https://user-images.githubusercontent.com/1294454/171301950-b4f586aa-13a3-4e45-801a-50bc33821f34.png">